kani-compiler/src/kani_middle/transform/check_uninit/mod.rs

```diff
@@ -31,13 +31,8 @@ mod relevant_instruction;
 mod ty_layout;
 
 /// Trait that the instrumentation target providers must implement to work with the instrumenter.
-trait TargetFinder {
-    fn find_next(
-        &mut self,
-        body: &MutableBody,
-        bb: BasicBlockIdx,
-        skip_first: bool,
-    ) -> Option<InitRelevantInstruction>;
+pub trait TargetFinder {
```

Comment:
why do you need a trait?

Reply:
There are two so-called target finders: one for finding regular uninitialized memory places to instrument and one for delayed UB. Both of them cause the exact same instrumentation to be added (though in different places). Instrumentation is handled by `UninitInstrumenter` struct, so to avoid duplicating the code I decided to parametrize it over `TargetFinder`, hence the trait.

Reply:
ok... thanks